packages/flutter_tools/lib/src/commands/doctor.dart

```diff
@@ -14,6 +14,12 @@ class DoctorCommand extends FlutterCommand {
       negatable: false,
       help: 'Run the Android SDK manager tool to accept the SDK\'s licenses.',
     );
+    argParser.addFlag('verbose',
+      abbr: 'v',
+      defaultsTo: false,
+      negatable: false,
+      help: 'Display all diagnostics related to Flutter, not just those with problems.',
+    );
```

Comment:
We shouldn't add a verbose here. We should use the global one. See `lib/executable.dart`; just pass verboseHelp to the `DoctorCommand` as a `verbose` argument or some such (the way `hidden` and `verboseHelp` are done).

Reply:
Done. I've tried to make the extension of that idea look as good as I could.